torch/testing/_internal/common_methods_invocations.py

```diff
@@ -968,6 +985,19 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad):
            sample_inputs_func=sample_inputs_fliplr_flipud,
            test_inplace_grad=False,
            supports_tensor_out=False),
+    OpInfo('linalg.slogdet',
+           aten_name='linalg_slogdet',
+           op=torch.linalg.slogdet,
+           dtypes=floating_and_complex_types(),
+           test_inplace_grad=False,
+           supports_tensor_out=False,
+           sample_inputs_func=sample_inputs_slogdet,
+           output_func=itemgetter(1),
+           decorators=[skipCUDAIfNoMagma, skipCPUIfNoLapack],
+           skips=(
+               # These tests do not work with output_func=itemgetter(1)
+               # TODO: remove this once https://github.com/pytorch/pytorch/issues/49326 is resolved
+               SkipInfo('TestCommon', 'test_variant_consistency_jit'),)),
```

Comment:
While we're here let's remove the slogdet entries in method_tests:

https://github.com/pytorch/pytorch/blob/4a3a37886cdbdca108536a2730c034db4f96f095/torch/testing/_internal/common_methods_invocations.py#L2295

slogdet is just a deprecated alias for linalg.slogdet at this point.